haskell-src/db-schema/migrations/2.2.0.1_swap_signers_pkey.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE signers DROP CONSTRAINT signers_pkey;
```

Comment:
We should also change the primary key in `haskell-src/lib/ChainwebDb/Types/Signer.hs`

Reply:
Nice catch!

Reply:
Funny enough, the Haskell code already looks like it's declaring the first column of the primary key to be `requestkey`: https://github.com/kadena-io/chainweb-data/blob/b976ae53b4584a900a63e579b8deec53c53bd4b1/haskell-src/lib/ChainwebDb/Types/Signer.hs#L77

Seems like `beam` is being very haphazard here.